lib/registrant.rb

```diff
@@ -0,0 +1,24 @@
+class Registrant
+  attr_reader :name, :age, :permit, :license_data
+  attr_accessor :permit
+  def initialize(name, age, permit = false)
+    @name   = name
+    @age    = age
+    @permit = permit
+    @license_data = {:written => false, 
+                     :license => false, 
+                     :renewed => false}
+  end
+
+  def earn_permit
+    @permit = true
+  end
+
+  def permit?
+    if @permit == false
+      false
+    else
+      true
+    end
```

Comment:
This works, but what if all that was inside this method was `@permit`? Then if @permit was true, it would just return true. If @permit was false, it would return false. So no need for the if block in this one.